torch/_lazy/__init__.py

```diff
@@ -20,3 +20,27 @@ def wait_device_ops(devices=None):
     if devices is None:
         devices = []
     torch._C._lazy._wait_device_ops(devices=devices)
+
+def sync_multi(tensors, devices):
+    """
+    Sync the list of lazy tensors so there IR get lowered for the activate backend
+    and the compiled computation graph get cached.
+    """
+    torch._C._lazy._sync_multi(tensors, devices)
+
+def get_tensor_id(tensor):
+    """Return a unique id of the lazy tensor maintained by LTC"""
+    return torch._C._lazy._get_tensor_id(tensor)
+
+def get_tensors_text(tensors):
```

Comment:
i wonder if we want to make a _lazy.debug. module for apis like this,
and if we want to rename the APIs

get_tensors_text -> debug.dump_ir(tensors)?

get_tensors_dot -> debug.render_ir_graph(tensors)?

it's easier to land it as-is, i am ok with possibly renaming/moving in a separate PR.

Reply:
And for get_tensors_backend , rename to debug.dump_ir_backend(tensors) ?

I can do that in a followup PR

Reply:
yea, sgtm.  (or, just make a parameter to debug.dump_ir(tensors, format="text"/"backend")? which is cleaner?